CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log for coreJSON Library
 
+## v3.2.0 (August 2022)
```

Comment:
The new version here is `v3.2.0` however in all other changes its `v3.1.1`.

Reply:
Fixed this, also did the "MISRA compliance updates" - > "MISRA C:2012  compliance updates." that you had marked in corePKCS11.